src/layout/Group/GroupSummary.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Heading, Paragraph } from '@digdir/designsystemet-react';
+import cn from 'classnames';
+
+import classes from 'src/layout/Group/GroupSummary.module.css';
+import type { LayoutNode } from 'src/utils/layout/LayoutNode';
+
+type GroupComponentSummaryProps = {
+  componentNode: LayoutNode<'Group'>;
+  childComponents: JSX.Element[] | null;
+  hierarchyLevel?: number;
+};
+
+export const GroupSummary = ({ componentNode, childComponents, hierarchyLevel = 0 }: GroupComponentSummaryProps) => {
+  const title = componentNode.item.textResourceBindings?.title;
+  const description = componentNode.item.textResourceBindings?.description;
+  const headingLevel = 3 + hierarchyLevel <= 6 ? 3 + hierarchyLevel : 6;
```

Comment:
Vanskelig å lese logikken her, kanskje legge det inn i en funksjon?

Signaturen kan f eks være:

`const getLevel = (): HeadingProps['level'] => 1;`

Da bør du kunne slippe castingen på linje 24 under også.

Reply:
Godt poeng 👍 Lager en funksjon for denne